sdk/core/azure-core/azure/core/tracing/__init__.py

```diff
@@ -8,5 +8,6 @@
     HttpSpanMixin,
```

Comment:
I might be missing some context, so want to check - I thought this entire implementation would go into https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/core/azure-core-tracing-opentelemetry, not azure-core directly. Why did we change course?

Reply:
I was wondering that also, maybe it is accident, will check with Marko once he is back from OOF

Reply:
My suggestion was `azure-core-tracing-opentelemetry`